docs/tutorial/frontend-tut.md

```diff
@@ -384,6 +386,8 @@ Here are some of those restrictions again, along with pointers about how to lift
     That example is rather brittle: it takes exactly 16 inputs, banked into four arrays, and adds their values together.
     Try incorporating this brittle version into your MrXL-to-Calyx compiler at first, and you can later think about generalizing it to any (commutative) operation, memories of any length, and any parallelism factor.
 
+    If you'd like to read more about reduction trees, you can check out [these slides][reduc-trees] from David Kirk and Wen-mei W. Hwu.
```

Comment:
Will this not make a broken link?

Reply:
sorry, why might that be the case? when i test with `mdbook serve`the link works!

Reply:
Ah great. I just worried because I was looking only at the diff created by this PR, and from that PoV it seems like you have cited a link that does not yet exist. All good, please disregard!

Reply:
yeah i know right? it was funny that the link was already there haha